src/interfaces/new-request-interface.ts

```diff
@@ -17,6 +17,7 @@ export interface NewRequestIF {
   assumedNameOriginal: string
   conditionsModalVisible: boolean
   conflictId: string
+  continuationInError: boolean
```

Comment:
In an alternate universe, we'd have named this `errorContinuationIn` (and the others similarly) to keep them together when sorted alphabetically.

Reply:
Warped us into that alternate universe Sev 😎 Made this change.

Reply:
Haha, nice 🚀